src/lib/entropy/entropy_srcs.cpp

```diff
@@ -83,6 +87,24 @@ class Processor_RNG_EntropySource final : public Entropy_Source {
 
 #endif
 
+#if defined(BOTAN_HAS_JITTER_RNG)
+
+class Jitter_RNG_EntropySource final : public Entropy_Source {
+   public:
+      size_t poll(RandomNumberGenerator& rng) override {
+         const size_t poll_bits = BOTAN_RNG_RESEED_POLL_BITS;
+         rng.reseed_from_rng(m_rng, poll_bits);
+         return poll_bits;
+      }
+
+      std::string name() const override { return "jitter_rng"; }
```

Comment:
This should just forward to `m_rng.name()`

Reply:
Done.